packages/community-cli-plugin/src/utils/loadMetroConfig.js

```diff
@@ -61,6 +61,7 @@ function getOverrideConfig(
       // We can include multiple copies of InitializeCore here because metro will
       // only add ones that are already part of the bundle
       getModulesRunBeforeMainModule: () => [
+        ...(config.serializer?.getModulesRunBeforeMainModule('unused') ?? []),
```

Comment:
Can we instead pass through the `entryFilePath` arg? (please test)

```suggestion
      getModulesRunBeforeMainModule: (entryFile: string) => [
        ...(config.serializer?.getModulesRunBeforeMainModule(entryFile) ?? []),
```